src/components/cardbuilder/cardBuilder.js

```diff
@@ -1303,6 +1303,7 @@ function updateUserData(card, userData) {
         if (!countIndicator) {
             countIndicator = document.createElement('div');
             countIndicator.classList.add('countIndicator');
+            countIndicator.classList.add('indicator');
```

Comment:
I believe you could pass both classes in a single add call

Reply:
Yup - that works. Updated.